schemas/context/enduserids.schema.json

```diff
@@ -22,6 +22,11 @@
       "title": "Adobe Analytics Cloud End User Ids",
       "$ref": "https://ns.adobe.com/xdm/context/identity",
       "description": "Adobe Analytics Cloud extension to End User Ids."
+    },
+    "https://ns.adobe.com/experience/tntid": {
```

Comment:
Why do we use `tntid` instead of `target`? `tnt` either refers to an explosive or a product name that we stopped using more than five years ago.

Reply:
This is in a extension for a application and that is the name of the field that is currently in use and will remain in use until it fully migrates over to mcid. So in the context of that extension it is 100% accurate, though as you rightly point out it has suffered from naming evolution.

Reply:
if that's the common name used by the application we should keep it.  As @cdegroot-adobe it is appropriately namespaced.